### ISSUE TITLE
Multiple commands produce GoogleService-Info.plist

### ISSUE THREAD

**Issue by rutaba1** [lohanidamodar/flutter_flavors#2]
Hi! I'm facing this issue when I execute app on iOS

 Multiple commands produce '/Users/rutaba/Desktop/fielder_shadman/fielder-mobile-app/build/ios/Debug-dev-iphonesimulator/Fielder.app/GoogleService-Info.plist

Can you help me out on how to solve this one?

**Comment by 1pavanb** [comment]
@rutaba1 sucks to be us
did you find the solution?

**Comment by lohanidamodar** [comment]
@1pavanb @rutaba1 I am sorry I haven't been able to update this or maintain this as this was for a tutorial. However I have a updated starter project for flutter firebase with Flavors already properly setup and easy to clone and start building your own project with it. Please check https://github.com/lohanidamodar/flutter_firebase_starter

**Comment by 1pavanb** [comment]
@lohanidamodar I'm integrating this library into an existing project. No time to reverse engineer that project you show.
Thanks for your concern.
I figured out the solution.

I'm running a bit late today. I'll try to post the changes for fixing later.

**Comment by zanesc** [comment]
@1pavanb what was the solution?

**Comment by 1pavanb** [comment]
after I'm done with the tutorial, I see these two files are added to compile sources by default. Those need to be removed.

![image](https://user-images.githubusercontent.com/36474201/97838572-78492180-1d06-11eb-9bff-6f76e59c3a0f.png)
During compilation, it tries to copy both of them into the Runner folder just like it does for AppDelegate.m and GeneratedPluginRestraint.m

So the updated Xcode identifies this behaviour as bad. While looking for the solution some people suggest to choose legacy compiler in the workspace setting to fix this. (I don't like that 😕)

**TL;DR**
Delete both GoogleService-Info.plist from Compiled Sources. Above picture is what it looks like currently.

I'm writing this out of memory, don't know If I'm missing anything. Let me know if it works.
@zanesc

**Comment by lohanidamodar** [comment]
@1pavanb awesome info, I believe this is due to recent updates in the Xcode and flutter. It's time I update the tutorial :D

**Comment by zanesc** [comment]
@1pavanb it worked. thank you
